Adyen/Assets/Generated/LocalizationKey.swift

```diff
@@ -86,6 +86,8 @@ public struct LocalizationKey {
     public static let cardNumberItemUnsupportedBrand = LocalizationKey(key: "adyen.card.numberItem.unsupportedBrand")
     /// The entered card brand isn't supported
     public static let cardNumberItemUnknownBrand = LocalizationKey(key: "adyen.card.numberItem.unknownBrand")
+    /// Scan your card
```

Comment:
if you would merge develop into base branch and then into this one, this should go away

Reply:
Okay.